components/customs/filters/available-filters.tsx

```diff
@@ -336,6 +337,12 @@ export const useAvailableFilters = () => {
       label: "Serotype (DENV only)",
```

Comment:
Out of curiosity what is the purpose of this hook? It does not return anything or take any arguments so just wanted to understand.

Also see it has some Arbo only filters (and maybe sero only filters) togethere. Is that a good idea?

Reply:
1st question: This is a hook because I want to call other hooks inside it. In this case, `useContext`. I can't call a hook in a normal function so this is a hook too :)

2nd question: Yeah the danger here is that you can specify a filter for a field that doesn't actually exist in your estimate (Ex. `populationGroup` for SarsCov2). This is helpful for stuff like `whoRegion` though since both of these can share the same filter. My ideal solution here would be that you could only use a filter if your `TEstimate` has the filterable field as a key in the object so `whoRegion` is good for both arbo and sc2 but something like `pathogen` is not. We're not there quite yet though, I've got to lab it out a bit more :man_scientist: 

Does that kind of answer the questions?